### DISCUSSION TITLE
How to display the exploded view of parts in  a model?

### DISCUSSION THREAD

**Discussion by joylix** [google/model-viewer#4806] [Q&A] [answered]
Take a look at this example:   [drone](https://sketchfab.com/3d-models/drone-fur-leif-test-1-c4208da5782f41d39c770f30edba06bc) , How can I display the drop-down box and check exploded_view to display the animation as in the above example?   here is my html code: 
```
<!DOCTYPE html>
<html>
<head>
  <title>3D Model Viewer - Drone Exploded View</title>
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
 <script type="module" src="https://ajax.googleapis.com/ajax/libs/model-viewer/3.5.0/model-viewer.min.js"></script>
</head>
<body>
  <model-viewer  src="models/drone.glb" shadow-intensity="1" camera-controls touch-action="pan-y"></model-viewer>
</body>
</html>
```
Or I can use commands to control the display of exploded_view without showing the menu bar. How should I do it? Thank you for your help.

**Comment by danilokrulj** [comment, marked as answer]
Since it is just an animation you should be able to set it up following these instructions. You can also stop the animation if you need to have the exploded view shown.

https://modelviewer.dev/examples/animation/index.html
